lib/agent/ocsp_response_cache.js

```diff
@@ -23,16 +23,16 @@ const status = {
 };
 
 // validate input
-const capacity = GlobalConfig.getOcspResponseCacheMaxSize();
 const maxAge = GlobalConfig.getOcspResponseCacheMaxAge();
 
-Errors.assertInternal(Util.number.isPositiveInteger(capacity));
 Errors.assertInternal(Util.number.isPositiveInteger(maxAge));
 
 const cacheDir = GlobalConfig.mkdirCacheDir();
 const cacheFileName = path.join(cacheDir, "ocsp_response_cache.json");
-// create a cache to store the responses
-const cache = new SimpleCache({maxSize: capacity});
+// create a cache to store the responses, dynamically changes in size
+var cache;
```

Comment:
Removed the initialization here since it's now based on:
- the response cache's size or 
- the response cache's size + previous cache's size - # of overlap between the two caches